utility/src/main/java/org/oppia/android/util/parser/image/UrlImageParser.kt

```diff
@@ -304,11 +305,18 @@ class UrlImageParser private constructor(
             drawableWidth *= multipleFactor
           }
         }
-        val drawableLeft = if (imageCenterAlign) {
+
+        fun isRTLMode(): Boolean {
+          return ViewCompat.getLayoutDirection(htmlContentTextView) == ViewCompat
+            .LAYOUT_DIRECTION_RTL
+        }
```

Comment:
Could you move this function outside of computeBounds(), but  keep it still within the BlockImageTarget class? The function could also be private to the class.